package.json

```diff
@@ -27,7 +27,7 @@
     "async": "^2.1.5",
     "debug": "^2.1.1",
     "loopback-connector": "^4.0.0",
-    "loopback-oracle-installer": "~3.0.0",
+    "oracledb": "^2.0.0",
```

Comment:
For the platforms where pre-built binaries don't install this could be a problem.  To keep the node-oracledb install infrastructure small (and avoiding security issues by not using 3rd party tools like node-pre-gyp), there is no fallback to compile from source.  To build from source the package specifier has to pull from GitHub.

Reply:
Oh, that's good to know but bad to hear that there is no fallback to build from source as part of npm install.

Reply:
It was just too heavyweight with too much potential for malicious injection.

Reply:
But solutions are welcome.